ukpa/models.py

```diff
@@ -0,0 +1,34 @@
+from django.conf import settings
+from django.db import models
+from django.shortcuts import get_object_or_404
+from django.urls import reverse
+
+from ironcage.utils import Scrambler
+from ironcage.validators import validate_max_300_words
+
+
+class Nomination(models.Model):
+
+    nominee = models.OneToOneField(settings.AUTH_USER_MODEL, related_name='nomination', on_delete=models.CASCADE)
+    statement = models.TextField(validators=[validate_max_300_words])
+
+    created_at = models.DateTimeField(auto_now_add=True)
+    updated_at = models.DateTimeField(auto_now=True)
+
+    id_scrambler = Scrambler(3000)
```

Comment:
Could you use 5000 here?  We're already using 3000 for CFP proposals.  The argument to `Scrambler()` is a kind of seed, and by reusing the seed we'd be reusing IDs -- not the end of the world, but something I'd like to avoid.